passkey-client/src/lib.rs

```diff
@@ -367,6 +364,45 @@ where
             client_extension_results: AuthenticatorExtensionsClientOutputs::default(),
         })
     }
+
+    fn map_rk(&self, criteria: &Option<AuthenticatorSelectionCriteria>) -> bool {
+        let supports_rk = self.authenticator.get_info().options.is_some_and(|o| o.rk);
```

Comment:
**Suggestion:** This method doesn't need to call `get_info`. It's already done in the `register` method, the result should just be passed down to this method instead.

Reply:
Fixed!